packages/plugins/other/visitor-plugin-common/src/base-resolvers-visitor.ts

```diff
@@ -880,6 +876,82 @@ export class BaseResolversVisitor<
     return `Array<${t}>`;
   }
 
+  protected createResolversUnionTypes(): Record<string, string> {
```

Comment:
`createResolversUnionTypes` is the simplified version of `createResolversFields` targeted for Unions. 
I chose to create a new function because `createResolversFields` is quite large and rigid so it's hard to implement/maintain